examples/mongodbatlas_project/main.tf

```diff
@@ -11,4 +11,15 @@ resource "mongodbatlas_project" "test" {
     name  = "atlas.project.deployment.nodesPerPrivateLinkRegion"
     value = 3
   }
+  tags = {
+    Owner       = "Terraform"
+    Environment = "Example"
+    Team        = "tf-experts"
+  }
+
+  lifecycle {
+    ignore_changes = [
+      tags["CostCenter"] # useful if `CostCenter` is managed outside terraform
+    ]
+  }
 }
```

Comment:
new line at the end

Reply:
fixed in 35d729e4e186219cab55b274cb1af334b82e1645